package.json

```diff
@@ -9,24 +9,25 @@
   },
   "dependencies": {
     "@ginger.io/jay-z": "0.0.8",
-    "aws-sdk": "^2.640.0",
+    "aws-sdk": "^2.760.0",
```

Comment:
The AWS deps are the main thing of interest here, but I upgraded others while I was here.